app/views/shared/_representation_order_details.html.haml

```diff
@@ -0,0 +1,11 @@
+= govuk_table do |table|
+  - table.with_caption(size: 's', text: 'Details from MAAT (beta)')
```

Comment:
Will this still be in beta?

Reply:
This will not actually be displayed yet as this PR is just to get the connection code in the codebase, so that it can be used and built on. I think, however, that when we get to the point of displaying information it should be indicated that it is beta or experimental for a period of time.